tools/seurat_qc.cwl

```diff
@@ -6,7 +6,7 @@ doc: "Run custom QC on 10X output"
 requirements:
   - class: ShellCommandRequirement
   - class: DockerRequirement
-    dockerPull: "pgc-images.sbgenomics.com/d3b-bixu/scrna_qc:v1.0.1"
+    dockerPull: "pgc-images.sbgenomics.com/brownm28/scrna_qc:v1.1.0"
```

Comment:
Is this the correct docker or should we move back to the d3b repo?

Reply:
It's a chicken and the egg thing, once I merge the 1.1 docker PR, I can update this before merging this guy.